docs/getting_started/requests.rst

```diff
@@ -0,0 +1,157 @@
+ImageResources
+==============
+
+Reading images isn't always limited to simply loading a file from a local disk.
+Maybe you are writing a web application and your want to read images from HTTP,
+or your images are already in memory as a BytesIO object. Maybe you are doing
+machine learning and decided that it is smart to compress your images inside a
+ZIP file to reduce the IO bottleneck. All these are examples of different
+places where image data is stored (aka. resources). 
+
+ImageIO supports reading (and writing where applicable) for all of the above
+examples and more. To stay organized, we group all these sources/resources
+together and call them ``ImageResource``. Often ImageResources are expressed as
+URIs though sometimes (e.g., in the case of byte streams) they can be python
+objects, too.
+
+Here, you can find the documentation on what kind of ImageResources ImageIO
+currently supports together with documentation and an example on how to
+read/write them.
+
+
+Files
+-----
+
+Arguably the most common type of resource. You specify it using the path to the
+file, e.g. ::
+
+    img = iio.imread("path/to/my/image.jpg")  # relative path
+    img = iio.imread("/path/to/my/image.jpg")  # absolute path on Linux
+    img = iio.imread("C:\\path\\to\\my\\image.jpg")  # absolute path on Windows
+
+
+Notice that this is a convenience shorthand (since it is so common).
+Alternatively, you can use the full URI to the resource on your disk ::
+
+    img = iio.imread("file://path/to/my/image.jpg")
+    img = iio.imread("file:///path/to/my/image.jpg")
+    img = iio.imread("file://C:\\path\\to\\my\\image.jpg")
+
+
+Byte Streams
+------------
+
+ImageIO can directly handle (binary) file objects and BytesIO objects. This includes
+subclasses of the above as well as generic objects that implement ``close`` and a ``read`` and/or ``write`` function.
```

Comment:
I got a bit confused by the "and" in "file objects and BytesIO objects", because a BytedIO object is a binary file object, right? So maybe something like:
```suggestion
ImageIO can directly handle (binary) file objects. This includes ``BytesIO`` objects (and subclasses thereof)
as well as generic objects that implement ``close`` and a ``read`` and/or ``write`` function.
```